playbooks/uda-workflow/connector-install/install_sap_hana_plugin.yml

```diff
@@ -0,0 +1,68 @@
+# => Cohesity Agent Management
+# =>
+# => Collection: cohesity.dataprotect
+# =>
+
+# => Install the Cohesity Datastore Agent on each SapHana host
+# => specified in the Ansible inventory
+# =>
+---
+- hosts: saphana
+  # => We need to gather facts to determine the OS type of
+  # => the machine
+  gather_facts: true
+  vars:
+    state: "present"
+    platform: SapHana
+    saphana_admin: sssadm
```

Comment:
Is this the same in our documentation ? if not just give this as sap_hana_user

Reply:
Give a generic users/password/home instead of specific

Reply:
Updated.